### PR TITLE
Ensure all TODOs are associated with an issue

### PR DESCRIPTION
**Pull request by daviddrysdale**
# Checklist

- [x] Pull request affects core Oak functionality (e.g. runtime, SDK, ABI)
  - [ ] I have written tests that cover the code changes.
  - [ ] I have checked that these tests are run by [Cloudbuild](cloudbuild.yaml)
  - [ ] I have updated [documentation](docs/) accordingly.
  - [ ] I have raised an [issue](https://github.com/project-oak/oak/issues) to
        cover any TODOs and/or unfinished work.
- [ ] Pull request includes prototype/experimental work that is under
      construction.

### CODE REVIEW

**Comment by daviddrysdale** [comment]
Some new issues, some pre-existing issues, and also moved storage to use the generated gRPC client helpers (which in turn required a new parameter for the code generator)

File: scripts/check_todo [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+readonly SCRIPTS_DIR="$(dirname "$(readlink -f "$0")")"
+# shellcheck source=scripts/common
+source "$SCRIPTS_DIR/common"
+
+check_todo() {
+  local path="$1"
+  local result
+  result=$(grep --with-filename --line-number TODO "$path" | grep --invert-match --regexp='TODO(#[0-9][0-9]*)')
+  if [ -n "$result" ]; then
```
Comment (tiziano88) [inline scripts/check_todo]: Should this use `[[` too?

File: third_party/grpc-rust/protoc-rust-grpc/src/lib.rs [hunk 2]
```diff
@@ -26,6 +26,8 @@ pub struct Args<'a> {
     pub rust_protobuf: bool,
     /// Customize rust-protobuf codegen
     pub rust_protobuf_customize: protoc_rust::Customize,
+    /// Crate name for oak crate; defaults to "oak" if empty.
+    pub oak_crate: &'a str,
```
Comment (tiziano88) [inline third_party/grpc-rust/protoc-rust-grpc/src/lib.rs]: This feels quite hacky. Also, shouldn't the generated code be part of `oak_abi`, rather? And perhaps the log pseudo-node may also get a similar treatment, i.e. a gRPC server definition (with a single method on it). This does seem to go towards the direction of having the ABI more and more specified via an IDL, similar to how Fuchsia does modules (e.g. https://fuchsia.dev/reference/fidl/fuchsia.logger) -- which I support! But perhaps we should think that through some more and figure out how to best tackle the issue as a whole? ref #426

File: scripts/check_todo (hunk 1 above)
Comment (daviddrysdale) [reply scripts/check_todo]: Done.

File: third_party/grpc-rust/protoc-rust-grpc/src/lib.rs (hunk 2 above)
Comment (daviddrysdale) [reply third_party/grpc-rust/protoc-rust-grpc/src/lib.rs]: Dropped this and raised a separate issue (#757) for now, to unblock this PR.

**Pull request merged**